### PR TITLE
Feature/gulp update 2

### PR DESCRIPTION
**Pull request by jjbert18**
if this doesn't work see Garrett 😉

### CODE REVIEW

File: source/package.json [hunk 1]
```diff
@@ -1,50 +1,31 @@
 {
-  "name": "subtheme",
-  "description": "Subtheme file for us in KIT-related projects",
+  "name": "unitedrentals",
+  "description": "UR: Drupal 8 KIT common theme support",
```
Comment (spfaffly) [inline source/package.json]: Probably change the name and description back to something KIT related.

Comment (jjbert18) [reply source/package.json]: good catch! sorry that got moved in

File: source/package.json [hunk 2]
```diff
@@ -2,49 +2,28 @@
   "name": "subtheme",
   "description": "Subtheme file for us in KIT-related projects",
   "author": "",
-  "version": "8.1.2",
+  "version": "1.0.0",
```
Comment (garrettrathbone) [inline source/package.json:5]: I feel like we should make this 8.2.0, to follow convention that it's made for Drupal 8. Similarly, I was thinking I'd make our new tools be 8.2 as well, to make sure that we have a deprecated tool set for all of the current sites.

File: source/gulpfile.js [hunk 3]
```diff
@@ -1,263 +1,219 @@
...
-        .pipe(isBuild() ? gulpUtil.noop() : changed(config.javascript.dest))
-        .pipe(isProd() ? gulpUtil.noop() : sourcemaps.init())
-        .pipe(isProd() ? uglify() : gulpUtil.noop())
-        .pipe(isProd() ? gulpUtil.noop() : sourcemaps.write('./maps'))
-        .pipe(gulp.dest(config.javascript.dest)));
-    }
+function buildImages() {
+  const tasks = config.map(function(entry) {
+    return src('themes/custom/' + entry.themeName + '/images/**/*')
+      .pipe(changed(entry.destDir + '/img'))
```
Comment (vml-kkedrovsky) [inline source/gulpfile.js:110]: Should this be the directory in the "dest" task below?

**Comment by garrettrathbone** [comment]
Closed since Karl's PR has this included.

**Pull request closed without merging**